packages/opencensus-instrumentation-http/README.md

```diff
@@ -5,6 +5,62 @@ OpenCensus HTTP Instrumentation allows the user to automatically collect trace d
 
 This project is still at an early stage of development. It's subject to change.
 
+## Installation
+
+Install the opencensus-instrumentation-http package with NPM:
+```bash
+npm install @opencensus/instrumentation-http
+```
+
+## Usage
+
+#### Let's create HTTP server!
+We'll need to require the ```http``` module and bind our server to the port 3000 to listen on.
+
+```javascript
+// content of index.js
+const http = require('http');
+const port = 3000;
+
+// A function which handles requests and send response
+const requestHandler = (request, response) => {
+  console.log(request.url);
+  response.end('Hello Node.js Server!');
+}
+
+// Creates a server
+const server = http.createServer(requestHandler);
+
+// Starts the server !
+server.listen(port, (err) => {
+  if (err) {
+    throw err;
+  }
+
+  console.log(`Node HTTP listening on ${port}`);
+});
+```
+
+#### Now, set up a HTTP Instrumentation.
+Now that we have our app working, let’s actually add the HTTP Trace instrumentation.
+
+```javascript
+const { plugin } = require('@opencensus/instrumentation-http');
```

Comment:
It's not clear to me where this code should go from the text of this readme alone. Before the `index.js` content listed above? After it? Another file?

Reply:
I'm also a little concerned that this is not really using the APIs in the "recommended" way (through the `Tracing` singleton). While I know it's to highlight how to enable just this single plugin, I'm not sure what value this example is providing for anyone that might land on this README file.

I think this example would be better moved to a separate location if we need it, and the README should focus on common usage patterns only - how a developer would most generally need to interact with the library. In this case "not much" since it's enabled by default when using the Tracing singleton.